apps/web/app/(use-page-wrapper)/settings/(settings-layout)/SettingsLayoutAppDirClient.tsx

```diff
@@ -201,7 +201,7 @@ const useTabs = ({ isDelegationCredentialEnabled }: { isDelegationCredentialEnab
 
         // TODO: figure out feature flag as it doesnt cause a re-render of the component when loaded.
         // You have to refresh the page to see the changes.
```

Comment:
@hbjORbj do you think this comment is still valid? do I need to test anything regarding this? or safe to remove this comment?